README.md

```diff
@@ -536,6 +537,38 @@ We have a small collection of algorithms, one for every day of the month. Scroll
   - **Resources:**
       - [Support File (Exchange Rates.csv)](src/docs/Dec21-Exchange_Rates.csv)
 
+
+## **December 22 - Huffman Encoding**
+  - **Problem**
+    - Huffman coding is a lossless data compression algorithm. The idea is to assign variable-length codes to input characters, lengths of the assigned codes are based on the frequencies of corresponding characters. The most frequent character gets the smallest code and the least frequent character gets the largest code.
+    - Write a program to generate Huffman codes if a set of characters and their corresponding frequency is given.
+
+
+  - **Example**
```

Comment:
Give an explanation of why each letter gets that code. Be specific. For example, why is f = 0 but the next most frequent character, e = 111? Why not 1 or 01 or 10 or anything else?

Also as a side note, for the remaining 8 questions, I think we can go back to the easier ones which are straightforward programming stuff.